eng/pipelines/coreclr/templates/build-perf-maui-apps-net6.yml

```diff
@@ -27,7 +27,7 @@ steps:
 
   # Get the current maui nuget config so all things can be found and darc based package sources are kept up to date.
   - script: |
-      curl -o NuGet.config 'https://raw.githubusercontent.com/dotnet/maui/main/NuGet.config'
+      curl -o NuGet.config 'https://raw.githubusercontent.com/dotnet/maui/net6.0/NuGet.config'
```

Comment:
Do you actually want the `net7.0` branch here? It has 6.0.11 feeds:

https://github.com/dotnet/maui/blob/net7.0/NuGet.config

Reply:
When I tested this, the net6.0 NuGet config also allowed the apps to build, if this turns out to not be the case I will make another PR to use net7.0.